experimental/displaynames/src/options.rs

```diff
@@ -48,11 +67,12 @@ pub enum Style {
     Short,
     Long,
     Menu,
+    None,
```

Comment:
Suggestion: `Option<Style>` and resolve the default in the individual formatter

Reply:
Makes sense. Added the change.